### PR TITLE
Translated: 'counter-cache' and 'translate'

### PR DESCRIPTION
**Pull request by tavaresgerson**

### CODE REVIEW

File: pt/orm/behaviors/translate.rst [hunk 1]
```diff
@@ -1,12 +1,476 @@
...
+            $this->addBehavior('Translate', [
+                'fields' => ['title'],
+                'validator' => 'translated'
+            ]);
+        }
+    }
+
+O texto acima usará o validador criado por ``validationTranslated`` para entidades traduzidas validadas.
+
+.. versionadded:: 3.3.0
```
Comment (saeideng) [inline pt/orm/behaviors/translate.rst]: you are writing docs for 4.x , no need to point to any version added for 3.x series

File: pt/orm/behaviors/counter-cache.rst [hunk 2]
```diff
@@ -1,12 +1,136 @@
...
 
-    Por favor, sinta-se a vontade para nos enviar um *pull request* para o
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **IMPROVE THIS DOC** para propor suas mudanças diretamente.
+    O comportamento do CounterCache funciona apenas para associações ``belongsTo``. 
+    Por exemplo, para "Comentários pertence a artigos", é necessário adicionar o 
+    comportamento do CounterCache ao ``CommentsTable`` para gerar ``comment_count`` 
+    para a tabela Articles.
+
+    It is possible though to make this work for ``belongsToMany`` associations.
```
Comment (othercorey) [inline pt/orm/behaviors/counter-cache.rst]: You left the english here.

File: pt/orm/behaviors/translate.rst [hunk 3]
```diff
@@ -1,12 +1,476 @@
...
+comportamento às suas necessidades, continue lendo o restante deste capítulo.
+
+Inicializando a Tabela do Banco de Dados i18n
+=============================================
+
+Para usar o comportamento, você precisa criar uma tabela ``i18n`` com o 
+esquema correto. Atualmente, a única maneira de carregar a tabela ``i18n`` 
+é executando manualmente o seguinte script SQL no seu banco de dados:
+
+.. code-block:: sql
```
Comment (othercorey) [inline pt/orm/behaviors/translate.rst]:
```suggestion
.. code-block:: mysql
```

File: pt/orm/behaviors/translate.rst [hunk 4]
```diff
@@ -1,12 +1,476 @@
...
+---------------------------------------
+
+Se você deseja usar uma tabela diferente de ``i18n`` para converter um repositório específico, 
+pode especificar o nome da classe da tabela para sua tabela personalizada na configuração 
+do comportamento. Isso é comum quando você tem várias tabelas para traduzir e deseja uma separação 
+mais limpa dos dados armazenados para cada tabela diferente::
+
+    class ArticlesTable extends Table
+    {
+
```
Comment (othercorey) [inline pt/orm/behaviors/translate.rst]: Remove extra line.

File: pt/orm/behaviors/translate.rst [hunk 5]
```diff
@@ -1,12 +1,476 @@
...
+Os registros de tradução podem conter qualquer sequência, se um registro tiver sido traduzido e 
+armazenado como uma sequência vazia (''), o comportamento da conversão será usado e o substituirá 
+pelo valor do campo original.
+
+Se isso for indesejável, você pode ignorar as traduções vazias usando a chave de configuração 
+``allowEmptyTranslations``::
+
+    class ArticlesTable extends Table
+    {
+
```
Comment (othercorey) [inline pt/orm/behaviors/translate.rst]: Remove extra line.

File: pt/orm/behaviors/translate.rst (hunk 1 above)
Comment (othercorey) [reply pt/orm/behaviors/translate.rst]: All of these still exist.  Can leave it for the translation.

File: pt/orm/behaviors/counter-cache.rst (hunk 2 above)
Comment (tavaresgerson) [reply pt/orm/behaviors/counter-cache.rst]: Sorry :-)

File: pt/orm/behaviors/translate.rst (hunk 1 above)
Comment (tavaresgerson) [reply pt/orm/behaviors/translate.rst]: Items before translated too?

**Pull request merged**